components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -114,6 +114,10 @@
                 "additionalProperties": false
             }
         },
+        "terminationGracePeriodSeconds": {
```

Comment:
In the protocol it's marked as optional, does it need to be marked optional here too?

Reply:
By making this change, all requests will now be able to set this despite it not being actually implemented by workspaces. Is that intended? Do we want to control release of this in some way?

Reply:
> In the protocol it's marked as optional, does it need to be marked optional here too?

it is optional by default

Reply:
Ah, it used to have `optional` in the v2 version of the spec. in `v4` it only has `required`. Thanks!